src/library/scala/Array.scala

```diff
@@ -55,6 +55,19 @@ object Array {
    */
   def newBuilder[T](implicit t: ClassTag[T]): ArrayBuilder[T] = ArrayBuilder.make[T](t)
 
+  /** Builds an array from the iterable collection
+   *
+   *  {{{
+   *  scala> Array.from(Seq(1,5))
+   *  val res75: Array[Int] = Array(1, 5)
+   *
+   *  scala> Array.from(Range(1,5))
+   *  val res76: Array[Int] = Array(1, 2, 3, 4)
+   *  }}}
+   *
+   *  @param  it the iterable collection
+   *  @return    an array consisting of elements of the iterable collection
+   */
   def from[A : ClassTag](it: IterableOnce[A]): Array[A] = it match {
     case it: Iterable[A] => it.toArray[A]
```

Comment:
No one has commented on this unconscionable shadowing. I realize this is a doc PR, but it's the sort of thing that one ought to have strong opinions about. Is it forgivable because it's obviously an alias?